src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -160,33 +161,42 @@ static ucs_status_t uct_rdmacm_cm_id_to_dev_addr(struct rdma_cm_id *cm_id,
         ucs_assert(!memcmp(&cm_id->route.addr.addr.ibaddr.dgid,
                            &qp_attr.ah_attr.grh.dgid,
                            sizeof(qp_attr.ah_attr.grh.dgid)));
+        params.flags    |= UCT_IB_ADDRESS_PACK_FLAG_GID_INDEX;
+        params.gid_index = qp_attr.ah_attr.grh.sgid_index;
     }
 
+    ucs_debug("cm_id %p: ah_attr %s", cm_id,
+              uct_ib_ah_attr_str(ah_attr_str, sizeof(ah_attr_str),
+                                 &qp_attr.ah_attr));
+    ucs_assert_always(qp_attr.path_mtu != 0);
+    params.flags   |= UCT_IB_ADDRESS_PACK_FLAG_PATH_MTU;
+    params.path_mtu = qp_attr.path_mtu;
```

Comment:
assert always qp_attr.path_mtu!=0

Reply:
asserted just 2 lines above to keep code better aligned, will update with new constant